<?xml version="1.0"?>
<launch>

	<arg name="laser_enabled" default="false"/>
  <arg name="kinect_enabled" default="false"/>

	<!-- <env name="GAZEBO_MODEL_PATH" value="${GAZEBO_MODEL_PATH}:$(find desafio)/model"/>
  	<env name="GAZEBO_RESOURCE_PATH" value="${GAZEBO_RESOURCE_PATH}:$(find desafio)/model"/> -->


  <!-- these are the arguments you can pass this launch file, for example paused:=true -->
  
  <arg name="urdf_extras"         default="$(find desafio)/urdf/teste.xacro"/>
  
  <include file="$(find bir_cimatec4_map)/launch/gazebo.launch">
    <!-- <arg name="world_name" value="$(arg world_name)"/>  -->
    <arg name="paused" value="false"/>
    <arg name="use_sim_time" value="true"/>
    <arg name="gui" value="true"/>
    <arg name="headless" value="false"/>
    <arg name="debug" value="false"/>
    <arg name="verbose" value= "false"/>
  </include>

  <include file="$(find husky_gazebo)/launch/spawn_husky.launch">
      <arg name="laser_enabled" value="$(arg laser_enabled)"/>
      <arg name="kinect_enabled" value="$(arg kinect_enabled)"/>
      <arg name="urdf_extras" value="$(arg urdf_extras)"/>

  </include>

  <node pkg="gazebo_light_sensor_plugin" name="opencv_camera" type="vision.py" output="screen" />
  <!-- <node pkg="gazebo_light_sensor_plugin" name="robot_vision" type="controlvision.py" output="screen" /> -->

</launch>
